robottelo/cli/org.py

```diff
@@ -168,15 +168,3 @@ def remove_user(cls, options=None):
         """Removes an user from an org"""
         cls.command_sub = 'remove-user'
         return cls.execute(cls._construct_command(options))
-
-    @classmethod
-    def set_parameter(cls, options=None):
-        """Create or update parameter for an organization."""
-        cls.command_sub = 'set-parameter'
-        return cls.execute(cls._construct_command(options))
-
-    @classmethod
-    def delete_parameter(cls, options=None):
```

Comment:
Perhaps you should remove them in a separate refactoring PR , what do you think?